### ISSUE TITLE
Train MS COCO error

### ISSUE THREAD

**Issue by BunnyShan** [zhreshold/mxnet-ssd#90]
I am training MS COCO using the latest ssd version. But Train-CrossEntropy and Train-SmoothL1 become to 'nan' after a few batch. I found the valid count is about 10-20 each batch.
Get any advice on this?Thanks!

**Comment by AlanNewImage** [comment]
I met the same problem when training VOC07/12 ,with only 2 categories,person and car,and my training data has the same 2 categories.

**Comment by BunnyShan** [comment]
@AlanNewImage 
I follow the owner's instruction to train VOC07+12 and it works fine. I'm figuring what is the difference between these.

**Comment by BunnyShan** [comment]
Problem solved, i made a mistake with coco image label format. Sorry if someone read this.

**Comment by BunnyShan** [comment]
Problem solved, i made a mistake with coco image label format. Sorry if someone read this.

**Comment by ogail** [comment]
@BunnyShan what's the label format issue?  I've updated the labels, class numbers and created train and val rec files accordingly still getting nans.  Could u elaborate on what u changed?

**Comment by BunnyShan** [comment]
@ogail
I didn't normalize the bbox's coordinates, then the ground truth is screwed.You can check your .lst file to make sure your groud thruth bbox coordinates is in the correct format.

**Comment by ogail** [comment]
@BunnyShan interesting, how come the default bounding boxes for ms-coco is not normalized by default.  Does something like what's here (https://github.com/mapbox/rasterio/issues/525) close to what u did with bbox normalization?

**Comment by BunnyShan** [comment]
@ogail  just normalize the coordinates with image width and height, make it between 0 and 1. :)

**Comment by qxin** [comment]
@BunnyShan  hi,  I trained coco using this mxnet SSD but found that the mAP ( iou>0.5) is about 0.15 , but caffe ssd achieved 0.46 (iou>0.5). How was your experiment? Thanks.
